packages/contracts/test/utils/exchange_wrapper.ts

```diff
@@ -219,6 +219,10 @@ export class ExchangeWrapper {
         const isCancelled = await this._exchange.cancelled.callAsync(orderHashHex);
         return isCancelled;
     }
+    public async getOrderEpochAsync(makerAddress: string, takerAddress: string): Promise<BigNumber> {
+        const orderEpoch = new BigNumber(await this._exchange.orderEpoch.callAsync(makerAddress, takerAddress));
```

Comment:
Is there a reason to convert to BigNumber here? As far as I remember all contract wrappers already return our BigNumbers